spec/requests/pseuds_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe "Pseuds", type: :request do
+  subject { page }
+
+  describe "show" do
+    let(:user) { FactoryBot.create(:user, login: "Agrajag") }
+
+    before do
+      FactoryBot.create(:pseud, user: user, name: "Zaphod")
+      FactoryBot.create(:pseud, user: user, name: "Slartibartfast")
+      FactoryBot.create(:pseud, user: user, name: "Betelgeuse")
+    end
+
+    it "shows only the configured maximum number of pseuds" do
+      allow(ArchiveConfig).to receive(:ITEMS_PER_PAGE).and_return(3)
+
+      visit "/users/Agrajag/pseuds"
+
+      within("ol.pagination") do
```

Comment:
This test is failing because of the 2 subnav sections:
```
Capybara::Ambiguous:
  Ambiguous match, found 2 elements matching visible css "ol.pagination"
```